project/src/main/java/caselab/controller/users/payload/UserUpdateRequest.java

```diff
@@ -0,0 +1,14 @@
+package caselab.controller.users.payload;
+
+import jakarta.validation.constraints.Size;
+import lombok.Builder;
+
+@Builder
+public record UserUpdateRequest(
+    @Size(max = 50, message = "Display name must not exceed 50 characters")
```

Comment:
тут должно быть что-то типа:
```suggestion
    @Size(max = 50, message = "{user.update.request.display_name.invalid_size}")
```
А в message.properties поместить такую строчку:
`user.update.request.display_name.invalid_size=Display name must not exceed 50 characters`